nibabies/workflows/bold/resampling.py

```diff
@@ -72,17 +93,33 @@ def init_bold_surf_wf(mem_gb, surface_spaces, medial_surface_nan, name="bold_sur
     workflow.__desc__ = """\
 The BOLD time-series were resampled onto the following surfaces
 (FreeSurfer reconstruction nomenclature):
-{out_spaces}.
+{out_spaces}
 """.format(
-        out_spaces=", ".join(["*%s*" % s for s in surface_spaces])
+        out_spaces=", ".join(["*%s*" % s for s in surface_spaces]),
     )
 
+    if project_goodvoxels:
+        workflow.__desc__ += """\
+Before resampling, a "goodvoxels" mask [@hcppipelines] was applied,
+excluding voxels whose time-series have a locally high coetfficient of
+variation, or that lie outside the cortical surfaces, from the
+surface projection.
+"""
+
     inputnode = pe.Node(
         niu.IdentityInterface(
-            fields=["source_file", "subject_id", "subjects_dir", "t1w2fsnative_xfm"]
+            fields=[
+                "source_file",
+                "subject_id",
+                "subjects_dir",
+                "t1w2fsnative_xfm",
+                "anat_giftis",
+                "t1w_mask",
```

Comment:
Not sure if this is T1w specific, but one goal for down the line is to allow T1/T2 only processing, so to future-proof may want to rename this to `anat_mask`